app/views/admin/rebate_forms/_artefact.haml

```diff
@@ -1,89 +1,94 @@
-
 %header.main-header
   .container
     %p.heading
-      2017/2018 Rates Rebate
+      ="#{Rails.configuration.rating_year.to_i-1}/#{Rails.configuration.rating_year.to_i} Rates Rebate"
```

Comment:
should use the rating year on the rebate_form database record, not the one in the server config.